python/src/aiconfig/scripts/aiconfig_cli.py

```diff
@@ -46,14 +50,33 @@ def run_subcommand(argv: list[str]) -> Result[str, str]:
         LOGGER.debug("Running edit subcommand")
         res_edit_config = core_utils.parse_args(main_parser, argv[1:], EditServerConfig)
         LOGGER.debug(f"{res_edit_config.is_ok()=}")
-        res_servers = res_edit_config.and_then(_run_editor_servers)
         out: Result[str, str] = result.do(
             #
             Ok(",".join(res_servers_ok))
             #
-            for res_servers_ok in res_servers
+            for edit_config in res_edit_config
+            for cli_config in res_cli_config
+            for res_servers_ok in _run_editor_servers(edit_config, cli_config.config_path)
         )
         return out
+    elif subparser_name == "rage":
+        LOGGER.debug("Running rage subcommand")
+        print("Raging...")
+        rage.spin(2)
+        print("Please hold. Your call is important to us.\nA representative will be with you shortly.")
+        rage.spin(3, type="music")
+        print("Turning up the heat...")
+        rage.spin(2)
+        print("Looking for your flipping server logs...")
+        rage.spin(4, type="music")
+        print("Do you know how many Zuckerbucks I should be charging for this?")
+        rage.spin(5)
+        print("I'm glad we're finally spending time together.")
+        rage.spin(4, type="music")
+
+        res_rage_config = core_utils.parse_args(main_parser, argv[1:], rage.RageConfig)
+        res_rage = res_rage_config.and_then(rage.rage)
```

Comment:
Why are we making `rage.rage()` a class method which requires us to create a `res_rage_config` instead of using a static method? Is it becuase we want to re-use state/object in case user calls multiple rages in the same session?

Reply:
>  we want to re-use state/object

I would never do such a thing - how dare you.

it's just a module with a function. rage module has rage function. 

`def rage(config: RageConfig) -> Result[None, str]:
`